### ISSUE TITLE
Scrolling is not working

### ISSUE THREAD

**Issue by ymohdriz** [computerjazz/react-native-draggable-flatlist#106]
Hi team,

Thanks for a wonderful project.
Everything is working except scrolling. It would be great if someone provide an example with scroll working.

I believe the problem related to differentiating onLongPress and scroll?

Thanks,
Mohammed

**Comment by computerjazz** [comment]
What do you mean "scrolling is not working"? Can you provide an example?

**Comment by tfe2014** [comment]
> What do you mean "scrolling is not working"? Can you provide an example?

Ik think he means that inside a <ScrollView> the list is not scrollable. So if you have a long list of items you can't reach the bottom. I have the same issue, looking for a solution

**Comment by awe79** [comment]
I looked into it a bit and found a possible solution. The issue is the PanGestureHandler. If it's enabled={true} the parent ScrollView is not working. So maybe if we make it enabled true only before we start draggin or something end false again at the end of dragging. I tried this a bit but I could not get it 100% stable.

**Comment by computerjazz** [comment]
So this is issue is with nested vertical scrollable elements? I.e. a DraggableFlatlist inside a parent scrollview?

**Comment by awe79** [comment]
> So this is issue is with nested vertical scrollable elements? I.e. a DraggableFlatlist inside a parent scrollview?

Exactly. When I remove the ScrollView the vertical scroll from the DraggableFlatlist kicks in, but I would like to use the scroll from the parent ScrollView which seems to be blocked by the PanGestureHandler

**Comment by Soto92** [comment]
Any solution? I noticed that the problem is with multi touch enabled. If there was a way to disable the scroll view touch while I am moving an item from the list it would be good.

**Comment by computerjazz** [comment]
closing in favor of https://github.com/computerjazz/react-native-draggable-flatlist/issues/184 as it has the issue template filled out
